.github/workflows/test-command.yml

```diff
@@ -65,131 +65,10 @@ jobs:
       - name: Install Pyenv
         run: python3 -m pip install virtualenv==16.7.9 --user
       - name: Write Integration Test Credentials
-        run: ./tools/bin/ci_credentials.sh
+        run: ./tools/bin/ci_credentials.sh test ${{ github.event.inputs.connector }}
         env:
-          AMAZON_SELLER_PARTNER_TEST_CREDS: ${{ secrets.AMAZON_SELLER_PARTNER_TEST_CREDS }}
-          AMAZON_ADS_TEST_CREDS: ${{ secrets.AMAZON_ADS_TEST_CREDS }}
```

Comment:
did you create all the new secrets in GSM?

Reply:
No, it's not needed. I think all developers will migrate connectors' secrets during their new  enhancements and bugfixes gradually. GSM secrets are more prioritized for CI but if some new GSM secret is skipped, the CI script will use old github values.
Certainly we can copy all values to GSM but I'm not sure about their relevance because their source of truth is LastPass where now there is a complete mess

Reply:
No need to do it as part of this PR